x-pack/plugins/security/public/views/management/roles_grid/components/roles_grid_page.tsx

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiIcon,
+  EuiIconTip,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import _ from 'lodash';
+import React, { Component } from 'react';
+import { toastNotifications } from 'ui/notify';
+import { Role } from '../../../../../common/model/role';
+import { isRoleEnabled } from '../../../../lib/role';
+import { RolesApi } from '../../../../lib/roles_api';
+import { ConfirmDelete } from './confirm_delete';
+import { PermissionDenied } from './permission_denied';
+
+interface Props {
+  intl: InjectedIntl;
+}
+
+interface State {
+  roles: Role[];
+  selection: Role[];
+  filter: string;
+  showDeleteConfirmation: boolean;
+  permissionDenied: boolean;
+}
+
+const getRoleManagementHref = (roleName?: string) => {
+  return `#/management/security/roles/edit${roleName ? `/${roleName}` : ''}`;
+};
+
+class RolesGridPageUI extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      roles: [],
+      selection: [],
+      filter: '',
+      showDeleteConfirmation: false,
+      permissionDenied: false,
+    };
+  }
+
+  public componentDidMount() {
+    this.loadRoles();
+  }
+
+  public render() {
+    const { permissionDenied } = this.state;
+
+    const pageContent = permissionDenied ? <PermissionDenied /> : this.getPageContent();
+
+    return <div className="mgtRolesListingPage">{pageContent}</div>;
+  }
+
+  private getPageContent = () => {
+    const { roles } = this.state;
+    const { intl } = this.props;
+    return (
+      <EuiPageContent className="mgtRolesListingPage__content">
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle>
+              <h2>
+                <FormattedMessage
+                  id="xpack.security.management.roles.roleTitle"
+                  defaultMessage="Roles"
+                />
+              </h2>
+            </EuiTitle>
+          </EuiPageContentHeaderSection>
+          <EuiPageContentHeaderSection>
+            <EuiButton data-test-subj="createRoleButton" href={getRoleManagementHref()}>
+              <FormattedMessage
+                id="xpack.security.management.roles.createRoleButtonLabel"
+                defaultMessage="Create new role"
```

Comment:
```suggestion
                defaultMessage="Create role"
```

Can you also change this on the Users page? Just remove "new"

<img src="https://d.pr/free/i/JTjOd1+" width="25%" />